### ISSUE TITLE
Block from liking issue. No excessive use. Just blocked when login via InstaPy

### ISSUE THREAD

**Issue by MastaMack** [InstaPy/InstaPy#6402]
Hello :)

The Bot works quite fine. Login works. But when it comes to liking I am blocked from the first like on. It just happens when I am login via InstyPy. When I am logging manually - even in the FireFoxBrowser, I have no issues. So probably Instagram detects that the login Process is automated.

1. Is there a fix for that issue?

2. Is it maybe possible to login manually and then let the Bot do the work to check if it's really about the automated login process?

**Comment by camillevingere** [comment]
Same problem on likes :o

**Comment by m-bonanno** [comment]
Same 😢
 --> Image...
 --> Image was not able to get liked! maybe blocked?
 --> Invalid Like Element!
```

**Comment by erwinbierens** [comment]
having the same issues for quite a while

**Comment by MANUJ243** [comment]
I am having the same issue, it says, your account has been temporarily locked.
![screenshot](https://user-images.githubusercontent.com/35228399/143222362-2b7c6f41-245d-4efe-9d5c-6abdcc1ad70d.jpeg)

**Comment by m-bonanno** [comment]
In my case it's because the `Like` action has been blocked... It looks like a ban wave has came 😢

![image](https://user-images.githubusercontent.com/46788528/143223100-8b1b50f5-f020-4f3c-bd10-34d15f86210f.png)

**Comment by AI-Guru** [comment]
Or did the HTML just change?

**Comment by m-bonanno** [comment]
> Or did the HTML just change?

I don't think so because the `Like` action properly works (I checked it manually watching the script in action).

**Comment by AI-Guru** [comment]
I would bet my retirement pension that the HTML has changed :D

**Comment by AI-Guru** [comment]
I lost my retirement pension. :D

Deleted InstaPy data and likes work again. I did this a couple of times.

Is there a way to fix this without removing ALL data?

**Comment by newdispatcher** [comment]
I am still doing it by removing logs and again login

**Comment by adrielkirch** [comment]
I liked and comment too many pictures, and I can't access my feed anymore, can't interact ... People be careful with excessive use of instapy or calling too much instagram API, you may be blocked for ever if you persist doing it. I will stop doing this .

**Comment by AI-Guru** [comment]
Is there a good best practice of the dosage of InstaPy activities?

**Comment by MastaMack** [comment]
Doing 300 / 350 Server calls hourly. It works fine. And I am liking on my own every hour after the bot paused due to reaching hourly limits. When I am about to like and comment a lot by my own I set the hourly server calls to 200 and I can like around 100 pics additionally after bot paused.
